streaming/vision/convert/base.py

```diff
@@ -5,7 +5,7 @@
 from torch.utils.data import Dataset
 from tqdm import tqdm
 
-from ...base import MDSWriter
+from streaming.base import MDSWriter
```

Comment:
do you have opinions on relative vs absolute imports? i used relative imports throughout new repo, should be uniform about this decision in any case...

Reply:
PEP 8 recommends absolute imports and composer also does the same.

Reply:
Sounds good! Thanks